packages/v4/patternfly-docs/content/design-guidelines/components/button/button.md

```diff
@@ -49,7 +49,7 @@ A primary button is the most prominent button on a page, used for the most impor
 
 _Example of a primary button inside a toolbar_
 
-<img src="./img/prim_ex_1.png" alt="Example of a primary button inside a toolbar" />
+<img src="./img/prim_ex_1.png" alt="Example of a primary button inside a toolbar" width="1159"/>
```

Comment:
This image looks good but a little blurry- can you double check that it was exported to 2x PNG, and not 1x?